docs/content/breaking-changes/breaking-changes.md

```diff
@@ -55,6 +55,7 @@ For more information, see
 ## Field-level breaking changes
 
 * <a name="resource-schema-field-removal-or-rename"></a>Removing or renaming a field 
+* <a name="resource-schema-field-addition-of-exactly-one-of"></a>Adding "ExactlyOneOf" to an existing field
```

Comment:
I'd suggest adding this right after `Making an optional field required or adding a new required field` since it's similarly making something newly required. Also maybe being a little more explicit - something like this?

```suggestion
* <a name="resource-schema-field-addition-of-exactly-one-of"></a>Adding an "ExactlyOneOf" constraint that causes one or more previously-optional fields to be required or conflict with each other
```